### PR TITLE
0.22 Changelog.

### PR DESCRIPTION
**Pull request by rjrjr**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+
+### Kotlin
+
+ * BREAKING: Safer BackStackScreen construction. (#809)
+ * BREAKING: Pass ContainerHints via setContentWorkflow. (#808)
+ * Add variance to StatelessWorkflow type parameters. (#790)
+ * BREAKING CHANGE: ViewRegistry is now a ContainerHint. (#770)
+ * Make `lifecyclOrNull` public. (#747)
+ * Give Worker.doesSameWorkAs a default implementation that just compares by concrete type. (#746)
+ * BREAKING: Remove deprecated `testRender` API. (#743)
```
Comment (zach-klippenstein) [inline CHANGELOG.md]: Nit: Organize all the breaking changes under a sub-header? We've used that sort of formatting in the past.

Comment (zach-klippenstein) [inline CHANGELOG.md]: Also this change is gonna make adoption pretty slow I think – was hoping to convert more internal tests before releasing this version :(

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,26 @@
...
+ * Make event sinks queued and reusable instead of throw after going stale. (#742)
+
+### Swift
+
+ * Add macOS support (#750)
```
Comment (zach-klippenstein) [inline CHANGELOG.md]: Nit: missing period
```suggestion
 * Add macOS support. (#750)
```

File: CHANGELOG.md (hunk 1 above)
Comment (rjrjr) [reply CHANGELOG.md]: Want me to revert it for this release?

Comment (rjrjr) [reply CHANGELOG.md]: Actually I just grep'd and there aren't all that many of them in our source, just ten files. I'll take a crack at converting.

**Comment by zach-klippenstein** [comment]
We decided to add a few more things to this release.

**Comment by zach-klippenstein** [comment]
This has gotten out of date, I will update once #835 lands and re-request reviews.

File: CHANGELOG.md (hunk 1 above)
Comment (zach-klippenstein) [reply CHANGELOG.md]: All internal tests have been converted, this is no longer a blocker.

**Comment by rjrjr** [comment]
The update LGTM. Go ahead and merge, @zach-klippenstein

**Pull request merged**